ecoscope/io/earthranger.py

```diff
@@ -272,17 +274,9 @@ def _get_observations(
         if observations.empty:
             return gpd.GeoDataFrame()
 
-        observations["created_at"] = pd.to_datetime(
-            observations["created_at"],
-            errors="coerce",
-            utc=True,
-        ).dt.tz_convert(tz)
-
-        observations["recorded_at"] = pd.to_datetime(
-            observations["recorded_at"],
-            errors="coerce",
-            utc=True,
-        ).dt.tz_convert(tz)
+        observations = clean_time_cols(observations)
+        observations["created_at"] = observations["created_at"].dt.tz_convert(tz)
+        observations["recorded_at"] = observations["recorded_at"].dt.tz_convert(tz)
```

Comment:
Do you think these conversions should also be part of `clean_time_cols`?

Reply:
So those fields are already run through `clean_time_cols` at this point in the code - these two lines are explicitly convert the times to the provided `tz` param

I'm not in love with the idea of adding the timezone conversion to `clean_time_cols` because a timezone value isn't parameterized in all the places where we want to call `clean_time_cols`
That said, I can see it being useful for `ecoscope-workflows` if we want to have a globally configured timezone preference, but that's possibly better as an explicit workflows task.

Happy to discuss this further, let me know what you think.